src/components/Assets/UploadAssetsComponent.jsx

```diff
@@ -0,0 +1,111 @@
+/* eslint-disable jsx-a11y/anchor-is-valid */
+/* eslint-disable react/prop-types */
```

Comment:
Why is this disabled?

Reply:
You didn't respond to my [question](https://github.com/AndelaOSP/art-dashboard/pull/177/files#r239448547)